.ci/Jenkinsfile

```diff
@@ -145,6 +153,10 @@ pipeline {
   }
   post {
     always {
+      // IMPORTANT: notify a build status in order to avoid GitHub checks with a wrong status.
+      whenTrue(matchesPrLabel(label: 'skip-ci')) {
+        error 'Pull Request has been configured to be disabled when there is a skip-ci label match'
```

Comment:
Out of curiosity, is not possible to report it as skipped instead of errord?

Reply:
If so, then GitHub checks will be passed

![image](https://user-images.githubusercontent.com/2871786/156185414-c30d1110-a6cb-4f55-9194-098cb9ef5fc5.png)

and as consequence, there is a possibility the PR could be accidentally merged without running everything in the CI.

I'm inclined to be explicit with an artificial failure to avoid the above if it makes any sense.